backend-api/src/functions/asyncActiveSession/asyncActiveSessionHandler.ts

```diff
@@ -23,6 +32,28 @@ export async function lambdaHandlerConstructor(
     return unauthorizedResponse;
   }
 
+  const jwe = authorizationHeaderResult.value;
```

Comment:
appreciate this is a jwe, but please may we align naming to the business terminology here:

`serviceToken`

I'm more than happy if we wanted to add some commentary talking about what this token is

Reply:
Updated